ambry-clustermap/src/main/java/com.github.ambry.clustermap/Datacenter.java

```diff
@@ -38,7 +38,7 @@
 
   private Logger logger = LoggerFactory.getLogger(getClass());
 
-  public Datacenter(HardwareLayout hardwareLayout, JSONObject jsonObject, ClusterMapConfig clusterMapConfig)
+  Datacenter(HardwareLayout hardwareLayout, JSONObject jsonObject, ClusterMapConfig clusterMapConfig)
```

Comment:
You could just ignore this, these are static cluster manager specific (added the doc). 
I did not make any significant changes to this class (just changed the access specifiers). Helix cluster manager will no longer use this class.